core/io/multiplayer_api.h

```diff
@@ -34,10 +34,15 @@
 #include "core/io/multiplayer_peer.h"
 #include "core/io/resource_uid.h"
 #include "core/object/ref_counted.h"
+#include "core/variant/typed_array.h"
+
+class MultiplayerReplicator;
 
 class MultiplayerAPI : public RefCounted {
 	GDCLASS(MultiplayerAPI, RefCounted);
 
```

Comment:
Is there a reason we make MultiplayerReplicator a friend class here and not just use proper encapsulation in MultiplayerReplicator? I tend to find the use if friend classes to be somewhat of a code smell.

Reply:
You are absolutely right, it stank so much :sweat_smile: . Fixed.